apps/web/src/app/(content)/changelog/page.tsx

```diff
@@ -1,17 +1,18 @@
-import { allChangelogs } from "contentlayer/generated";
-import { Rss } from "lucide-react";
-import type { Metadata } from "next";
-
-import { Button } from "@openstatus/ui";
 
+import type { Metadata } from "next";
 import {
   defaultMetadata,
   ogMetadata,
   twitterMetadata,
 } from "@/app/shared-metadata";
+import { Shell } from "@/components/dashboard/shell";
+import { Button } from "@openstatus/ui";
 import { Mdx } from "@/components/content/mdx";
 import { Timeline } from "@/components/content/timeline";
-import { Shell } from "@/components/dashboard/shell";
+import { allChangelogs } from "contentlayer/generated";
+import { Context } from "vm";
```

Comment:
No need for that import

Reply:
Do is it not need the Shell component? It was there before

Reply:
I was referring to `import { Context } from "vm";` 😀

Reply:
Oh yeah. lol Yeah that's gone